src/heroku.ts

```diff
@@ -37,4 +37,20 @@ export class HerokuClient {
       maxAttempts: 5,
     });
   }
+
+  getEnterpriseAccounts() {
```

Comment:
There are a ton of GitHub Actions warnings on this file's `any` uses. If a better type cannot be provided, I think it's good to add a lint exception and a bit of reasoning for future devs. Also, these functions need a return type declared, says the linter.